### ISSUE TITLE
Installer issues identifying system OS version

### ISSUE THREAD

**Issue by gdevdeiv** [serghey-rodin/vesta#1498]
### Operating System (OS/VERSION):

Debian 9.3

### VestaCP Version:

0.9.8-19

### Installed Software (what you got with the installer):

None, clean OS, haven't installed VestaCP or any other packages on top of the "clean" Debian installation.

### Steps to Reproduce:

You can't really reproduce it. It's more a conceptual issue than anything else.

So, I got a OVH dedicated server and installed Debian 9.3. The plan was to install Vesta and get things going, but guess what? Tried running the Vesta installation script and got this:

```
--2018-02-26 13:56:15--  http://vestacp.com/pub/vst-install-rhel.sh
Resolving vestacp.com (vestacp.com)... 104.236.66.100
Connecting to vestacp.com (vestacp.com)|104.236.66.100|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 48116 (47K) [text/plain]
Saving to: ‘vst-install-rhel.sh’

vst-install-rhel.sh 100%[===================>]  46.99K  --.-KB/s    in 0.1s

2018-02-26 13:56:15 (321 KB/s) - ‘vst-install-rhel.sh’ saved [48116/48116]

cut: /etc/redhat-release: No such file or directory
grep: /etc/redhat-release: No such file or directory
vst-install-rhel.sh: line 21: [: : integer expression expected
Error: No access to Vesta repository
```

Fair enough. It tries to fetch the RHEL installation script for some reason.. Debug time then: a quick look to the vst-install.sh file and 🎉

https://github.com/serghey-rodin/vesta/blob/master/install/vst-install.sh#L40

The installer is trying to "read" the OS version from the /etc/issue file, which isn't a good idea. Anyone with root privileges could edit that file and, basically, break the way Vesta detects the OS distribution and version; especially when your hosting company offers you an "optimized, tweaked kernel" and you find out that they modified that same file for some reason.

A good solution would be to do it the same way as in the vst-install-debian.sh file, https://github.com/serghey-rodin/vesta/blob/master/install/vst-install-debian.sh#L18, where we check for the /etc/os-release file, maybe even /etc/system-release for edge cases.

**Comment by gdevdeiv** [comment]
Anyone in need of a workaround until the issue is fixed can do this:

```
wget http://vestacp.com/pub/vst-install-debian.sh
bash vst-install-debian.sh
```

**Comment by dpeca** [comment]
I'm always using latest version on installer from github.
`wget https://raw.githubusercontent.com/serghey-rodin/vesta/master/install/vst-install-debian.sh`

**Comment by gdevdeiv** [comment]
@dpeca, then you should have no problems. This issue is only for the "global" installation script:

![a044cef78f73be841344e3c949cf59b9](https://user-images.githubusercontent.com/3106243/36678119-33127a12-1b10-11e8-904e-4421783e1df6.png)

**Comment by Skamasle** [comment]
This is a old bug with OVH 

You can read this from 2015

https://github.com/serghey-rodin/vesta/issues/1043
https://forum.vestacp.com/viewtopic.php?t=9993

**Comment by gdevdeiv** [comment]
I wouldn't call it a "bug", just a custom /etc/issue. Anyone with enough privileges in a Linux system can edit that file. That is just enough reason for me not to trust it will tell my actual OS version.

**Comment by Skamasle** [comment]
Yes, you are right but this only happen in ovh, no orther company or iso provider change it

This need some work to fix it
